pkg/pillar/hypervisor/kvm.go

```diff
@@ -852,6 +882,27 @@ func (ctx KvmContext) CreateDomConfig(domainName string, config types.DomainConf
 			}
 		}
 	}
+	if len(canBusAssignments) != 0 {
+		canIfContext := struct {
+			Machine    string
+			IfName     string
+			HostIfName string
+			ID         int
+		}{Machine: ctx.devicemodel, IfName: "", HostIfName: "", ID: 0}
+
+		t, _ = template.New("qemuCANBus").Parse(qemuCANBusTemplate)
+		id := 0
+		for canIf, canHostIf := range canBusAssignments {
+			fmt.Printf("CAN interface %s connected to host CAN %s\n", canIf, canHostIf)
```

Comment:
```suggestion
			log.Noticef("CAN interface %s connected to host CAN %s\n", canIf, canHostIf)
```

Reply:
log module is not used in this kvm.go, I prefer to keep as it is in other parts in the code and let this change to another PR....

Reply:
f.e. logrus is used in kvm.go: https://github.com/lf-edge/eve/pull/3802/files#diff-81e96cd930b1704b4214d18d7ebc150d7f17d3622878aaade9b4e1f2daa4a674L800

`fmt.Printf` does not show the filename of this log message, this makes searching logs for messages that come from `kvm.go` harder.

I fear those PRs will not come ...

Reply:
I agree using logrus is better than fmt.Printf

Reply:
Done.

Reply:
resolved